pkg/domain/dashboard/dashboard_test.go

```diff
@@ -142,3 +142,55 @@ func TestUIDExtraction(t *testing.T) {
 		})
 	}
 }
+
+func TestContentWithoutID(t *testing.T) {
```

Comment:
That's a lot of test for checking that the builtin `delete` function works. We don't need that IMO